src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -209,6 +212,10 @@ export class FileExplorerComponent implements OnInit {
     return path.substring(1, indexSelectedFolder) + selectedFolder;
   }
 
+  isResponseComplete() {
```

Comment:
It might be clearer to call this method something like `dataFinishedLoading()`, that would make the intent of the HTML clearer.